nodeadm/internal/aws/imds/imds.go

```diff
@@ -3,14 +3,22 @@ package imds
 import (
 	"context"
 	"io"
+	"time"
 
+	"github.com/aws/aws-sdk-go-v2/aws/retry"
 	"github.com/aws/aws-sdk-go-v2/feature/ec2/imds"
 )
 
 var client *imds.Client
 
 func init() {
-	client = imds.New(imds.Options{})
+	client = imds.New(imds.Options{
+		DisableDefaultTimeout: true,
+		Retryer: retry.NewStandard(func(so *retry.StandardOptions) {
+			so.MaxAttempts = 15
+			so.MaxBackoff = 1 * time.Second
```

Comment:
this means 1 second between each attempt, right?

Reply:
yea, the underlying exponential retryer just behaves linearly for 1 second backoff